### ISSUE TITLE
Error thrown when selected from the menu

### ISSUE THREAD

**Issue by DownesAyr** [yedidiaklein/moodle-local_aiquestions#5]
Hello I am not sure if the plugin is conflicting with some other plugins within our setup but it is causing an error and is essentially unusable at this point. The error occurs on any course I use to navigate to the plugin i.e. /local/aiquestions/story.php?courseid=1234

![image](https://github.com/yedidiaklein/moodle-local_aiquestions/assets/22812156/0a21b0ff-59f6-4102-bc61-db3275255d05)

From this I am guessing it is maybe conflicting with the badges pool plugin (another local plugin) but not sure. I have tried changing site theme and this does not resolve it either.

Hope you can help!

**Comment by wilenius** [comment]
Yeah it looks like somehow the way the plugins build the navigation tree are in conflict. Does the other plugin work normally? Does it also insert an item to the course navigation menu?

**Comment by DownesAyr** [comment]
It was the badge pool plugin that was the problem. It works fine but when you assign a badge to a category (which is what it lets you do) all courses within that category and sub-categories then throw the error when working with your plugin. It is an old plugin so that is probably why it was causing an issue. Appreciate your reply - I think ultimately the problem lies with their old plugin though. I disabled the badge pool and now your plugin is working great. Thanks!
